edge-agent/src/Microsoft.Azure.Devices.Edge.Agent.Kubernetes/CombinedKubernetesConfigProvider.cs

```diff
@@ -74,18 +92,16 @@ CombinedDockerConfig GetCombinedConfig(IModule module, IRuntimeInfo runtimeInfo)
 
         void MountSockets(IModule module, CreateContainerParameters createOptions)
         {
-            var workloadUri = new Uri(this.configSource.Configuration.GetValue<string>(Core.Constants.EdgeletWorkloadUriVariableName));
-            if (string.Equals(workloadUri.Scheme, "unix", StringComparison.OrdinalIgnoreCase))
```

Comment:
Error handling：
  - null and empty
- not a uri string
 - not unix

Reply:
it is handled on the top level when we initialize `KubernetesModule`. we will not start agent if these settings are unavailable.